src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/page.tsx

```diff
@@ -122,7 +120,14 @@ export default async function DashboardPage() {
       yearlySubscriptionUrl={yearlySubscriptionUrl}
       fxaSettingsUrl={fxaSettingsUrl}
       scanCount={scanCount}
-      totalNumberOfPerformedScans={totalNumberOfPerformedScans}
+      // Fallback to max scan threshold so we can’t exceed the limit
+      totalNumberOfPerformedScans={
+        profileStats?.total ??
+        parseInt(
+          process.env.NEXT_PUBLIC_ONEREP_MAX_SCANS_THRESHOLD as string,
+          10,
+        )
```

Comment:
Instead of artificially setting this to the highest number when we pass it in, might it not be clearer what's going on if we add `typeof props.totalNumberOfPerformedScans === "undefined" ||` to https://github.com/mozilla/blurts-server/blob/faeb16be71eadacfe3b8679e8190584d5312d2a0/src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/View.tsx#L316?

Reply:
Yeah, that does have the potential to lead to confusion — updated in [dd96d9e](https://github.com/mozilla/blurts-server/pull/3957/commits/dd96d9ea2ff4ee472e426e490e0150d88d569224).